src/utilities/matrices.jl

```diff
@@ -0,0 +1,5 @@
+# Funcs
+function lowertriangind(mat::Matrix)
```

Comment:
let's move this function to near where it's used and remove this file which only has one function